crates/ruma-client-api/src/room.rs

```diff
@@ -7,6 +7,9 @@ pub mod get_room_event;
 pub mod report_content;
 pub mod upgrade_room;
 
+#[cfg(feature = "unstable-msc3266")]
```

Comment:
We don't usually separate modules, even if they are unstable.

Reply:
What do you mean? There's a module per endpoint here, I've just followed what existed before.

Reply:
I mean the list is cut in two.